src/Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs

```diff
@@ -20,73 +20,41 @@ public sealed class SendMailMessage : PSCmdlet
         #region Command Line Parameters
 
         /// <summary>
-        /// Specifies the files names to be attached to the email.
+        /// Gets or sets the files names to be attached to the email.
         /// If the filename specified can not be found, then the relevant error
         /// message should be thrown.
         /// </summary>
         [Parameter(ValueFromPipeline = true, ValueFromPipelineByPropertyName = true)]
         [ValidateNotNullOrEmpty]
         [Alias("PsPath")]
         [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
-        public String[] Attachments
-        {
-            get { return _attachments; }
-            set
-            {
-                _attachments = value;
-            }
-        }
-        private String[] _attachments;
+        public string[] Attachments { get; set; }
 
         /// <summary>
-        /// Specifies the address collection that contains the
+        /// Gets or sets the address collection that contains the
         /// blind carbon copy (BCC) recipients for the e-mail message.
         /// </summary>
         [Parameter(ValueFromPipelineByPropertyName = true)]
         [ValidateNotNullOrEmpty]
         [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
-        public String[] Bcc
-        {
-            get { return _bcc; }
-            set
-            {
-                _bcc = value;
-            }
-        }
-        private String[] _bcc;
+        public string[] Bcc { get; set; }
 
         /// <summary>
-        /// Specifies the body (content) of the message.
+        /// Gets or sets the body (content) of the message.
         /// </summary>
         [Parameter(Position = 2, ValueFromPipelineByPropertyName = true)]
         [ValidateNotNullOrEmpty]
-        public String Body
-        {
-            get { return _body; }
-            set
-            {
-                _body = value;
-            }
-        }
-        private String _body;
+        public string Body { get; set; }
 
         /// <summary>
-        /// Specifies a value indicating whether the mail message body is in Html.
+        /// Gets or sets a value indicating whether the mail message body is in Html.
         /// </summary>
         [Parameter(ValueFromPipelineByPropertyName = true)]
         [Alias("BAH")]
-        public SwitchParameter BodyAsHtml
-        {
-            get { return _bodyashtml; }
-            set
-            {
-                _bodyashtml = value;
-            }
-        }
-        private SwitchParameter _bodyashtml;
+        public SwitchParameter BodyAsHtml { get; set; } = false;
```

Comment:
Is the init really needed?

Reply:
Not really needed. It would be false anyway. Just make it more explicit. Same for the other inits.
I've seen it in other cmdlets and adopted this style.